modules/gapi/include/opencv2/gapi/render/render.hpp

```diff
@@ -18,6 +18,56 @@
 #include <opencv2/gapi/own/exports.hpp>
 #include <opencv2/gapi/own/scalar.hpp>
 
+
+/** \defgroup gapi_draw G-API Drawing and composition functionality
+ *  @{
+ *
+ *  @brief Functions for in-graph drawing.
+ *
+ *  @note This is a Work in Progress functionality and APIs may
+ *  change in the future releases.
```

Comment:
and APIs may be changed in the future releases

Reply:
"may change" also works.

Reply:
"may change" also works.